py-polars/polars/dataframe/frame.py

```diff
@@ -5254,47 +5254,16 @@ def group_by_dynamic(
         Group based on a time value (or index value of type Int32, Int64).
 
         Time windows are calculated and rows are assigned to windows. Different from a
-        normal group by is that a row can be member of multiple groups. The time/index
-        window could be seen as a rolling window, with a window size determined by
-        dates/times/values instead of slots in the DataFrame.
+        normal group by is that a row can be member of multiple groups.
+        By default, the windows look like:
 
-        A window is defined by:
+        - [start, start + period)
+        - [start + every, start + every + period)]
+        - [start + 2*every, start + 2*every + period)]
+        - ...
 
-        - every: interval of the window
-        - period: length of the window
-        - offset: offset of the window
-
-        The `every`, `period` and `offset` arguments are created with
-        the following string language:
```

Comment:
the diff is kind of hard to read, but I'm moving all this "string language" part to the 'notes' section